vcd/resource_vcd_org.go

```diff
@@ -342,16 +362,16 @@ func resourceOrgUpdate(d *schema.ResourceData, m interface{}) error {
 
 	if err != nil {
 		log.Printf("[DEBUG] Error updating Org %s : %s", orgName, err)
-		return fmt.Errorf("error updating Org %s", err)
+		return diag.Errorf("error updating Org %s", err)
 	}
 	err = task.WaitTaskCompletion()
 	if err != nil {
 		log.Printf("[DEBUG] Error completing update of Org %s : %s", orgName, err)
-		return fmt.Errorf("error completing update of Org %s", err)
+		return diag.Errorf("error completing update of Org %s", err)
 	}
 
 	log.Printf("[TRACE] Org %s updated", orgName)
-	return nil
+	return diags
```

Comment:
Is there any reason we have defined `var diags diag.Diagnostics` and then use it in return (but nowhere else). Couldn't we simply return `nil`?

Reply:
Same as above! 😄